resources/css/time_line.css

```
@@ -178,3 +179,12 @@
 .header-post{
     background-color: white !important;
 }
+
+.submitEditComment{
```

```suggestion
.submitEditComment {
```
conventon này sửa hết đi em